lib/cli/index.js

```diff
@@ -53,7 +53,8 @@ function cli(options) {
   };
 
   var defaultUpdateCheckerOptions = {
-    checkForUpdates: true
+    checkForUpdates: true,
+    httpProxy: process.env['http_proxy']
```

Comment:
although the npm settings seem to be lower-case.  Is it possible for us to read these settings via npm as a lib?